addons/sale/report/sale_report_templates.xml

```diff
@@ -190,6 +190,7 @@
 <template id="report_saleorder_pro_forma">
     <t t-call="web.html_container">
         <t t-set="is_pro_forma" t-value="True"/>
+        <t t-set="docs" t-value="docs.with_context(proforma=True)"/>
```

Comment:
This seems like it shouldn't be there

Reply:
Why? and then where should it be?

Reply:
Because you are making this always proforma, for everyone, in every case?

Reply:
it is in the template with id="report_saleorder_pro_forma" then it should always be proforma, no?